.buildkite/pipeline.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+  - label: 'Test'
+    env:
+      SEGMENT_CONTEXTS: snyk, aws-credentials
+      SEGMENT_BUILDKITE_IMAGE: 'buildkite-agent-golang1.15'
```

Comment:
should we use golang1.16 here since go.mod is using 1.16?

Reply:
I changed it